app/src/main/java/com/globant/pokemontcgapp/util/MapperUtils.kt

```diff
@@ -0,0 +1,19 @@
+package com.globant.pokemontcgapp.util
+
+import com.globant.pokemontcgapp.R
+
+fun pokemonTypeImageMapper(type: String): Int =
```

Comment:
This File, maybe you can rename..because not give relevant information

Reply:
the File ...not the method

Reply:
Changed for a more descriptive name, thanks